### DISCUSSION TITLE
workaround for yarn install too slow under the GFW env

### DISCUSSION THREAD

**Discussion by howel52** [codesandbox/codesandbox-client#5317] [Show and tell]
`yarn install` is too slow under the GFW

![image](https://user-images.githubusercontent.com/9854818/102973379-93caee80-4537-11eb-8ff5-01c23b60870b.png)

Two step to solve this problem:
1. set registry in .yarnrc for npm packages

```
registry "https://registry.npm.taobao.org"
```

2. use bin-wrapper-china in package.json for dependent binaries
```diff
{
  ...
  "resolutions": {
+    "bin-wrapper": "npm:bin-wrapper-china"
  }
}
```

**Comment by SaraVieira** [comment]
Hey!

There is really not a lot we can do as the machines used to run containers are hosted in Europe and are not serverless based so they are not available in many different localtions
